molo/core/content_import/helpers/parse.py

```diff
@@ -12,6 +12,10 @@ def parse_validate_content(repos, locales):
     }
 
 
+def parse_import_content(repos, locales):
+    return parse_validate_content(repos, locales)
```

Comment:
I'm not sure why you need this.. is it a case of making it more readable? This feels like an alias.. :thought_balloon: